docs/content/version-reporting.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Privacy
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA periodically reports its version to a publicly hosted, external service. The version reports only contain the OPA
+version number (e.g., v0.12.3) and a randomly generated UUID. This feature is only applicable to the `opa run` and
+`opa version` commands.
+
+In case of the `opa run` command, this feature is **ON by-default** and can be easily disabled by specifying
+the `--incognito` flag. When OPA is started in either the server or repl mode, OPA calls the external service
+on a best-effort basis and shares the version it's running. The time taken to execute the remote call and process the
+subsequent response from the external service does not delay OPA's start-up.
+
+In case of the `opa version` command, this feature can be enabled by specifying the `--check` or `-c` flag.
+
+## External Service
+
+OPA uploads its version information by default at [telemetry.openpolicyagent.org](https://telemetry.openpolicyagent.org).
+The environment variable *OPA_TELEMETRY_SERVICE_URL* can be used to configure the external service OPA reports to.
```

Comment:
Let's add an example of the full HTTP request payload here. E.g.,

```http
POST /v1/report HTTP/1.1
Host: telemetry.openpolicyagent.org
Content-Type: application/json
User-Agent: <whatever OPA uses for the user-agent>
```
```json
{
  "id": "<Version 4 UUID generated when OPA starts>",
  "version": "v0.20.0-dev"
}
```

Reply:
Done